src/generic/index.ts

```diff
@@ -1,4 +1,17 @@
 import { Tensor } from '@tensorflow/tfjs-core';
+import 'seedrandom';
+
+
+function shuffle(inputs: Array<any>, seed?: string): void {
```

Comment:
```js
function seed(seedString: string): void;
function shuffle(inputs: Array<any>): void;
```
If we set the seed when call `shuffle`, it will be hard to control the random sequence, but if set before shuffle, it will be easy:
```js
generic.seed('my-seed');
let shuffled1 = generic.shuffle([ 1, 2, 3 ]);
let shuffled2 = generic.shuffle([ 4, 5, 6 ]);
```
or we can use it in the test case:
```js
generic.seed('my-seed');
let output = functionBeTested();
```
the output will be same whenever we call the function.